bin/lint.ts

```diff
@@ -0,0 +1,29 @@
+import { readdirSync } from 'fs';
+import { join } from 'path';
+
+const MAX_PARALLEL = Number(process.env.GAPI_MAX_PARALLEL) || 1;
+
+process.stdout.setMaxListeners(MAX_PARALLEL);
+process.stderr.setMaxListeners(1 + MAX_PARALLEL);
+
+const path = process.argv[2];
+const runAll = require('npm-run-all');
```

Comment:
Let's prefer `import * as runAll from 'npm-run-all';` ?
And it probably makes sense to put it in the beginning of the file, together with other imports.

Reply:
Agreed. I was having issues with the `import` syntax:

```ts
import runAll from 'npm-run-all';
```

for some unexplainable reason:

```
runAll([scripts.shift()], options) // run first synchronously to install TypeScript
      ^
TypeError: npm_run_all_1.default is not a function
    at Object.<anonymous> (/Users/namoscato/dev/git/google-api-typings-generator/bin/lint.ts:28:7)
```

but I can look into it.

Reply:
Yeah, that's probably because they don't have default export, so 
```js
import * as runAll from 'npm-run-all';
```
will work (`* as` part)

Reply:
_Right_ – thanks!